app/services/hyrax/custom_queries/find_by_model_and_property_value.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
```

Comment:
would be nice to have IIIF Print generator create this file for future applications but maybe that's a pipe dream